src/dash-table/dash/DataTable.js

```diff
@@ -731,6 +731,14 @@ export const propTypes = {
      */
     page_current: PropTypes.number,
 
+    /**
+     * `page_count` represents the number of the last page of the
```

Comment:
I understand page_count to be the number of pages of data in the table.
`the number of the last page` would be more like `page_max_index` (or page_count - 1), no?

```suggestion
     * `page_count` represents the number of the pages in the
```